app/components/BuyNBTC.tsx

```diff
@@ -43,25 +52,25 @@ function Instructions() {
 	);
 }
 
-interface FeeProps {
-	youReceive: number;
+interface TransactionStatusProps {
+	isSuccess?: boolean;
+	handleRetry: () => void;
 }
 
-function Fee({ youReceive }: FeeProps) {
+function TransactionStatus({ isSuccess, handleRetry }: TransactionStatusProps) {
 	return (
-		<Card className="p-4 bg-azure-10 rounded-2xl h-14">
-			<CardContent className="flex flex-col justify-between p-0">
-				<div className="flex justify-between gap-2">
-					<p className="text-gray-400">You Receive</p>
-					<p>{youReceive} nBTC</p>
-				</div>
+		<Card>
+			<CardContent className="p-6 rounded-lg text-white flex flex-col gap-4 bg-azure-10">
+				Transaction: {isSuccess ? "Success" : "Failed"}
+				<Button onClick={handleRetry}>Retry</Button>
```

Comment:
Lets rename it, its confusing that after a succesfull trade user can only press a `retry` button

Reply:
Renamed it to ok when it's success.